src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/store/ElasticSearchAnalysisGraph.java

```diff
@@ -104,9 +109,35 @@ public void construct() {
     hotNodeClusterRca.addTag(TAG_LOCUS, LOCUS_MASTER_NODE);
     hotNodeClusterRca.addAllUpstreams(Collections.singletonList(hotJVMNodeRca));
 
+    constructShardResourceUsageGraph();
+
     // constructResourceHeatMapGraph();
   }
 
+  private void constructShardResourceUsageGraph() {
+    Metric cpuUsage = new CPU_Utilization(5);
+    Metric ioTotThroughput = new IO_TotThroughput(5);
+    Metric ioTotSyscallRate = new IO_TotalSyscallRate(5);
```

Comment:
It is important that we reuse the metric nodes between RCAs or else we are making the SQLite queries again and again.

Reply:
`CPU_Utilization` is the node we are duplicating. The other node has `evaluationIntervalSeconds` as 1 second, while we are using 5 secs period in hot shard.

Reply:
The metrics are emitted every 5 seconds. Evaluation cannot be every 1 second. Its an oversight I think